nss-evening-cohort-24/hack-client-awareness-campaign
# CREATED A HOMEPAGE COMPONENT

## Description
- cleared default homepage text
- created a half-way hero welcome page
- created a hard-coded localhost link to creating a new campaign. will need to be changed when site is deployed.

## Related Issue
N / A - this was a stretch goal

## Motivation and Context
- Gives the user an introduction to the site and leads to the creation of a campaign

## How Can This Be Tested?
- Pull PR code
- Stand web server
- Stand local instance of DB
- it should be the first page you see on logon

## Screenshots (if appropriate):
N / A 

## Types of changes
